numpy/lib/function_base.py

```diff
@@ -3954,6 +4082,175 @@ def _lerp(a, b, t, out=None):
     return lerp_interpolation
 
 
+def _find_weighted_index(sk, qsn, interpolation='linear'):
+    dim = sk.shape # (N, d1, d2,..., dk)
+    Nx = dim[0]
```

Comment:
Use snake case. ex: `nx`

Reply:
The `_quantile_ureduce_func` which computes non-weighted quantiles terms sample size Nx.
Maybe it's okay to keep them with the same name?